stub/src/main/java/io/grpc/stub/ServerCalls.java

```diff
@@ -349,4 +349,31 @@ public void onCancel() {
     public void onComplete() {
     }
   }
+
+  public static void unimplementedStreamResponse(StreamObserver responseObserver) {
```

Comment:
IIRC, the reason of having one method for each call type was to have the flexibility of changing the implementation of one call type without affecting others. If so, we probably don't need to do the same for the unimplemented helpers because their "implementations" are too simple to allow any changes.